lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1137,6 +1138,22 @@ sub check_ecoscore_data($) {
 		}
 	}
 
+	# Extended Eco-Score data from impact estimator
```

Comment:
I think that would create a copy of the array content.

We could have my $data_quality_info_tags_ref = $product_ref->{data_quality_info_tags} and then push @{$data_quality_info_tags_ref}, '...'

but I'm not sure if it improves readabiltity or makes it worse. There's also consistency with the other functions of the module, and we push sometimes to data_quality_warning_tags, data_quality_error_tags, so it might be easier to not have to create a variable for each.

Reply:
ok, that make sense, here.